l5kit/l5kit/dataset/ego.py

```diff
@@ -98,18 +101,38 @@ def get_frame(self, scene_index: int, state_index: int, track_id: Optional[int]
             )
         data = self.sample_function(state_index, frames, self.dataset.agents, tl_faces, track_id)
 
+        # [history_num_frames + 1, 2]
```

Comment:
why are we computing this values here and not in the sampling function?

Reply:
can you help me to move it to sampling function?

Reply:
moved to sampling function